.github/workflows/ci.yml

```
@@ -87,13 +87,20 @@ jobs:
         with:
           command: test
           args: --no-run
-      - name: Run cargo test
+      - name: Run cargo test --all-features
         uses: actions-rs/cargo@v1
         env:
           RUST_TEST_THREADS: 1
         with:
           command: test
           args: --all-features
+      - name: Run cargo test --features rustls
+        uses: actions-rs/cargo@v1
+        env:
+          RUST_TEST_THREADS: 1
+        with:
+          command: test
+          args: --features rustls
```

This uses the default feature which includes the `rustls` 
```suggestion
      - name: Run cargo test
        uses: actions-rs/cargo@v1
        env:
          RUST_TEST_THREADS: 1
        with:
          command: test
```